wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1546,17 +1563,18 @@ extension SessionManager {
             return completion()
         }
 
-        switchingDelegate.confirmSwitchingAccount(completion: { confirmed in
+        switchingDelegate.confirmSwitchingAccount { confirmed in
             if confirmed {
                 activeUserSession.callCenter?.endAllCalls()
-                completion()
             }
-        })
+            completion()
+        }
```

Comment:
The behaviour changed, that `completion()` is now called in all cases, not only for `confirmed=true`. Is this intended?